contracts/MyToken.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import "openzeppelin-solidity/contracts/access/Ownable.sol";
+import "openzeppelin-solidity/contracts/security/Pausable.sol";
+
+contract MyToken is ERC20,Pausable,Ownable {
```

Comment:
You can use ready-made modules in legacy. In openzepelin there is a ready ERC20Pausable